### ISSUE TITLE
Not generating an exe?

### ISSUE THREAD

**Issue by FinlayDaG33k** [pyinstaller/pyinstaller#1461]
Pyinstaller does for some reason not generate an exe?
it showsup my scriptname in build, but it's not an exe (the extension isn't there, and when I rename it, it doesn't run properly)

the helloworld.py:
```
#!/usr/bin/python
import os
os.system("echo Hello World!")
```

**Comment by htgoebel** [comment]
Please read the manual, look at https://github.com/pyinstaller/pyinstaller/wiki/If-Things-Go-Wrong and https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs. Thanks.

**Comment by FinlayDaG33k** [comment]
I've read the documentation carefully, yet there is still no exe.
all dependencies are installed.

**Comment by htgoebel** [comment]
We still need *much* more information. "It doesn't work" is no point for starting to find out where there could be a problem.

**Comment by FinlayDaG33k** [comment]
I use command:
'pyinstaller --onefile helloworld.py -n helloworld.exe'

but the .exe is not able to run in windows, but is able to run in Linux
using the c9.io IDE to compile

**Comment by htgoebel** [comment]
PyInstaller does not support cross-freezing.

This was my last answer on this issue until you give *all* the information we request on https://github.com/pyinstaller/pyinstaller/wiki/How-to-Report-Bugs

**Comment by FinlayDaG33k** [comment]
Please make the documention so that people can actually do what why they use this script:
compile a python script to an windows exe script :)

**Comment by tallforasmurf** [comment]
See in the doc, third paragraph under this heading:
http://pythonhosted.org/PyInstaller/#overview-what-pyinstaller-does-and-how-it-does-it

The output of PyInstaller is specific to the active operating system and
> the active version of Python. To prepare a distribution for a different OS,
> or for a dfferent version of Python, you runPyInstaller on that OS, under
> that version of Python.
>

This is what Hartmut means by "PyInstaller does not support
cross-freezing." If you want a windows app, you must run PyInstaller on
Windows.

**Comment by FinlayDaG33k** [comment]
Weird, because every tutorial I follow says it will compile a windows exe even from the Linux environment since 1.4 :\

**Comment by FinlayDaG33k** [comment]
Managed to get it to work after fidling around with ActivePython and Pywin32 :)

**Comment by Ronaldomata34** [comment]
how you solved it?  I want to create .exe from .py, but in window not get .exe file? (I'm in ubuntu 16.04) It's per example like "program" if i try to click to execute not working :( Do somebody can to help me?

**Comment by tallforasmurf** [comment]
To make a Windows .exe file, run PyInstaller **in Windows**.

As the manual linked earlier explains, PyInstaller makes a bundle containing the Python that executes it, and the support libraries from the local OS. You do this in Ubuntu you will have a bundle containing the Ubuntu installed Python interpreter and all the local .so libs, plus a PyInstaller boot loader that was *compiled on Ubuntu* -- and none of them are compatible with Windows.

**Comment by heliocarbex** [comment]
Dear friends,

I'm using pyinstaller in Windows. I'm trying to generate a simple exe just printing "Hello world!".
When I try to run pyinstaller the executable file vanishes(the exe is being generated at the "build" folder). When I put the option "-d", the executable file is generated with all the error messages. Observing these messages I realized the python34.dll was missing. I just copied the dll from the "dist" folder to the "build" folder and it works! Someone knows why is this happening?
